### PR TITLE
[Blogs-1386] Ati Stats level 1 destination

### PR DESCRIPTION
**Pull request by tomwinnington**
Ticket: 
https://jira.dev.bbc.co.uk/browse/BLOGS-1385

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/ValueObject/AtiAnalyticsLabels.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     * @param string
+     */
+    private $appEnvironment;
+
+    public function setAppEnvironment(string $appEnvironment): void
+    {
+        $this->appEnvironment = $appEnvironment;
+    }
+
+    public function __construct(CosmosInfo $cosmosInfo)
```
Comment (benolot) [inline src/ValueObject/AtiAnalyticsLabels.php]: I think the constructor should be above "normal" public methods

File: src/ValueObject/AtiAnalyticsLabels.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    public function setAppEnvironment(string $appEnvironment): void
+    {
+        $this->appEnvironment = $appEnvironment;
+    }
+
+    public function orbLabels()
+    {
+        $labels = [
+            //'destination' => $this->getDestination(),
```
Comment (BjornTwachtmann) [inline src/ValueObject/AtiAnalyticsLabels.php]: Should this be commented? I thought setting this was the whole point of this ticket.

Comment (tomwinnington) [reply src/ValueObject/AtiAnalyticsLabels.php]: Sam said that destination isn't going live yet

**Pull request merged**